python/python/ert_gui/simulation/run_dialog.py

```diff
@@ -20,60 +39,52 @@
 class RunDialog(QDialog):
 
     def __init__(self, run_model, parent):
-        QDialog.__init__(self, parent)
+        QDialog.__init__(self, None)
```

Comment:
Why did you change this?

Reply:
The parent window flags caused strange window behaviour. Changing it to default window flag in and setting parent in next commit.